config/karma/pact.karma.conf.js

```diff
@@ -0,0 +1,48 @@
+/*jshint node: true*/
+'use strict';
+
+/**
+ * Requires the shared karma config and sets any local properties.
+ * @name getConfig
+ * @param {Object} config
+ */
+function getConfig(config) {
+  const minimist = require('minimist');
+  const argv = minimist(process.argv.slice(2));
+  require(`./${argv.watch ? 'watch' : 'test'}.karma.conf`)(config);
+
+  let skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(argv._[0]);
+  let testWebpackConfig = require('../webpack/test.webpack.config');
+  const logger = require('../../utils/logger');
+  const path = require('path');
+  const pactServers = require('../../utils/pact-servers');
+
+  skyPagesConfig.skyux.pactConfig.providers = pactServers.getAllPactServers();
```

Comment:
`skyPagesConfig.skyux` was designed to never be set from code - only set using the `skyuxconfig.json` files.  For things set in code, I would move them to `skyPagesConfig.runtime.X`.

Reply:
done